packages/zoe/test/unitTests/contracts/test-simpleExchange.js

```diff
@@ -291,18 +282,13 @@ test('simpleExchange showPayoutRules', async t => {
   });
 
   const alicePayments = { Asset: aliceMoolaPayment };
-  const { seat: aliceSeat1 } = await zoe.redeem(
-    aliceInvite,
-    aliceSale1OrderProposal,
-    alicePayments,
-  );
-
   // 4: Alice adds her sell order to the exchange
-  aliceSeat1.addOrder();
+  // TODO(msm): Capture the offerHandle here and use it in the test below.
```

Comment:
Sounds good!

Reply:
This is the open issue explained above.
Resolving as a dup of above conversation